src/utils/power_flow/make_pf.jl

```diff
@@ -52,15 +54,14 @@ function make_pf(system)
         bus_number = get_number(b)::Int
         bus_angle = get_angle(b)::Float64
         bus_voltage = get_voltage(b)::Float64
-        generator = nothing
+        generator = []
         for gen in get_components(Generator, system)
             if gen.bus == b
-                !isnothing(generator) && throw(DataFormatError("There is more than one generator connected to Bus $b.name"))
-                generator = gen
+                push!(generator, gen)
```

Comment:
This is a very bad operation to had in an iterative algorithm, basically each iteration you will be allocating these matrices.

Reply:
If that is a problem, it can be easily solved.  Create two variables to hold sum_active_power and sum_reactive_power, initialized to 0.0.  Instead of pushing to a vector, add each generator's powers.